rpc_jobs/periodic_cleanup.yml

```diff
@@ -40,64 +38,68 @@
         }
       }
 
-      // run node cleanups on all nodes
-      def nodes = getLongRunningNodes()
-      def parallel_steps = [:]
-      for (int i=0; i<nodes.size(); i++){
-        nodeName = nodes[i]
-        parallel_steps['node_'+nodeName] = {
-          common.use_node(nodeName){
-            stage("Cleanup "+nodeName){
-              dir("rpc-gating") {
-                sh "scripts/workspace_cleanup.sh"
-                sh "scripts/tmp_cleanup.sh"
-                sh "scripts/docker_cleanup.sh"
+      library "rpc-gating@${RPC_GATING_BRANCH}"
```

Comment:
Why was this line moved? There is a pattern in place for having this as the first line, unless there is a good reason to move this I think it should be left where it is.

Reply:
This was to match the [phobos](https://github.com/rcbops/rpc-gating/blob/master/rpc_jobs/phobos.yml#L91) job where the functions are defined first, then the library statement is still the first line to get executed. 

I'm happy to put the library line back at the top, but should probably change that job as well.

Reply:
Done